playground-nextjs/next.config.js

```diff
@@ -1,6 +1,6 @@
 /** @type {import('next').NextConfig} */
```

Comment:
All changes you see from this file and below are just formatting changes that were made programmatically by running `npx prettier . --write` (or `npm run prettier`).